test/gracenote/search_test.rb

```diff
@@ -0,0 +1,13 @@
+require "test_helper"
+
+module Gracenote
+  describe "#search" do
+    describe "with mode" do
+      it "should send mode" do
+        client = Gracenote::Client.new(client_id: "test", user_id: "test")
+        allow(client).to receive(:query).with("album_search", hash_including(mode: "SINGLE_BEST_COVER"), a_kind_of(Hash))
```

Comment:
__[rubocop]__
* Line is too long. [121/120] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)